internal/strings/strings.go

```diff
@@ -99,3 +99,39 @@ func InSlice(a string, list []string) bool {
 	}
 	return false
 }
+
+// FastLower returns a lower case version of the string
+// This function is faster than strings.ToLower
+func FastLower(s string) string {
```

Comment:
I think the name is misleading also we should add a reference https://cs.opensource.google/go/go/+/refs/tags/go1.19.3:src/strings/strings.go;l=600.

I think `asciiToLower` is a more appropriate name.